### ISSUE TITLE
Chapter 9. page 103, paragraph 3, confusing sentence

### ISSUE THREAD

**Issue by colwem** [crypto101/book#276]
> Usually, this value is either 3 or 65537; because those numbers have few 1 numbers in their binary expansion, you can compute the exponentiation more efficiently

Fewer than what? 3 is 2 digits long, 65537 is 17 digits long.  Where's the connection?

**Comment by lvh** [comment]
I'm not sure why that sentence is confusing; it says few, not fewer. 65537 has _few_ (specifically, 2) 1's in its binary expansion, making exponentiation fast. (3, too, has only 2 1's in its binary expansion.) It's not about how many digits long it is; that only has a minute impact on how long it takes to exponentiate.

**Comment by colwem** [comment]
Ok, wow I misread that. Though I think I misread that because it's kind of a clumsy way of saying that.  Grammatically (or stylistically) I think fewer just feels more natural and so my brain "fixed" it.  I would rephrase this to something like "because in binary those numbers don't have many 1 digits"  Specifically I think "not many" is better than "few" here.

**Comment by lvh** [comment]
OK; I've rewritten the sentence a little in the tagged commit; let me know if that helps.

**Comment by colwem** [comment]
Yeah, that's perfect.
